dynamodb_create_cloudwatch_alarms/main.py

```diff
@@ -8,40 +8,75 @@
 
 
 Usage:
-    dynamodb-create-cloudwatch-alarms [options]
+    dynamodb-create-cloudwatch-alarms (-s <sns_topic_arn>) [options]
     dynamodb-create-cloudwatch-alarms [-h | --help]
 
 Options:
-     --debug   Don't send data to AWS
+     -s <sns_topic_arn>    For sending alarm ( require )
+     -r <region>           AWS region
+     -p <prefix>           DynamoDB name prefix
+     --debug               Don't send data to AWS
 
 """
 import boto
 import boto.ec2
-import boto.dynamodb
+import boto.dynamodb2
 from docopt import docopt
 from boto.ec2.cloudwatch import MetricAlarm
 
 DEBUG = False
+AWS_REGION = u'ap-northeast-1'
```

Comment:
I'd rather not put any default. Make it so it uses the default region configured in boto, and if use the region specified if one is specified.